bedrock/firefox/redirects.py

```diff
@@ -231,4 +231,8 @@ def firefox_mobile_faq(request, *args, **kwargs):
 
     # bug 960543
     redirect(r'^firefox/(?P<vers>[23])\.0/eula', '/legal/eula/firefox-{vers}/'),
+
+    # bug 1224060
+    redirect(r'^ja/firefox/ios/(?P<vers>[0-9]+(\.[0-9]+)*)/(?P<file>releasenotes|system-requirements)',
+             'http://www.mozilla.jp/firefox/ios/{vers}/{file}/'),
```

Comment:
Thanks. 
Added local_prefix flag. 

www.mozilla.jp does not services https, so I did not change the destination URL.